toolchain/check/import_cpp.cpp

```diff
@@ -131,10 +139,146 @@ auto ImportCppFiles(Context& context, llvm::StringRef importing_file_path,
   auto name_scope_id = AddNamespace(context, package_id, imports);
   SemIR::NameScope& name_scope = context.name_scopes().Get(name_scope_id);
   name_scope.set_is_closed_import(true);
+  name_scope.set_is_cpp_scope(true);
+
+  *ast = std::move(generated_ast);
+  context.sem_ir().set_cpp_ast(ast->get());
 
   if (ast_has_error) {
     name_scope.set_has_error();
   }
 }
 
+static auto ClangLookup(Context& context, SemIR::NameId name_id)
+    -> std::optional<clang::LookupResult> {
+  clang::ASTUnit* ast = context.sem_ir().cpp_ast();
+  CARBON_CHECK(ast);
+  CARBON_CHECK(ast->hasSema());
+  clang::Sema& sema = ast->getSema();
```

Comment:
```suggestion
  clang::Sema& sema = ast->getSema();
```

Note the implementation of getSema:

```
  bool hasSema() const { return (bool)TheSema; }

  Sema &getSema() const {
    assert(TheSema && "ASTUnit does not have a Sema object!");
    return *TheSema;
  }
```

Reply:
Done.